sdk/maintenance/Microsoft.Azure.Management.Maintenance/src/Microsoft.Azure.Management.Maintenance.csproj

```diff
@@ -6,10 +6,10 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.Maintenance</PackageId>
     <Description>Microsoft Azure Maintenance Management Library</Description>
-    <Version>1.3.0</Version>
+    <Version>1.4.0-preview</Version>
```

Comment:
The version is inconsistent with the one defined in `AssemblyInfo.cs`. Consider to align with the one you defined in `AssemblyInfo.cs`.

Reply:
I think Assembly info accept Int32.Int32.Int32.Int32 format, Existing dotnet sdk for other RP followed same pattern for preview version, AssemblyInfo.cs file wont have preview string.

e.g.
https://github.com/Azure/azure-sdk-for-net/blob/b629fc67041c7f19193c40cce9cf8850b1bbe572/sdk/apimanagement/Microsoft.Azure.Management.ApiManagement/src/Microsoft.Azure.Management.ApiManagement.csproj#L11

https://github.com/Azure/azure-sdk-for-net/blob/b629fc67041c7f19193c40cce9cf8850b1bbe572/sdk/apimanagement/Microsoft.Azure.Management.ApiManagement/src/Properties/AssemblyInfo.cs#L13